SmartPlaces.Facilities/lib/OntologyMapper/src/MappingFileException.cs

```diff
@@ -19,6 +19,12 @@ public MappingFileException(string message, string filename)
             this.filename = filename;
         }
 
+        public MappingFileException(string message, string filename, Exception innerException)
+            : base(message, innerException)
+        {
+            this.filename = filename;
+        }
+
         public string Filename => filename;
     }
 }
```

Comment:
For consistency's sake, this file should move to `.\SmartPlace.Facilities\lib\OntologyMapper\src\Exceptions\MappingFileException.cs`

Reply:
Agree on principle, disagree in practice. The autogenerated docs don't deal with many namespaces well, it makes for a very crowded menu, so I'm thinking it'd be preferable to have as few namespaces in these packages as possible (but no less).

Reply:
We can adapt our pattern of matching the namespace to folder structure if its causing complications. We dont have to add `.Exceptions` but moving the folder structure would keep the project cleaner.

Reply:
OK, I guess I can live with that :-)